client/blocks/inline-help/contextual-help.js

```diff
@@ -81,6 +81,57 @@ const fallbackLinks = [
 	},
 ];
 
+const bloggerFallbackLinks = [
+	{
+		get link() {
+			return localizeUrl( 'https://wordpress.com/support/posts/' );
+		},
+		get title() {
+			return translate( 'All about blog posts' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'About Blog Posts' )` *ES Score: 6*